src/sentry/api/endpoints/organization_stats_summary.py

```diff
@@ -118,7 +120,7 @@ class StatsSummaryApiResponse(TypedDict):
 
 @extend_schema(tags=["Organizations"])
 @region_silo_endpoint
-class OrganizationStatsSummaryEndpoint(OrganizationEventsEndpointBase):
+class OrganizationStatsSummaryEndpoint(OrganizationEndpoint):
```

Comment:
Now that handle_query_errors has been decoupled from the OrganizationEventsEndpointBase, we can extend the OrganizationEndpoint instead